src/main/java/nextstep/fp/Lambda.java

```diff
@@ -25,32 +25,65 @@ public void run() {
             }
         }).start();
     }
+    
+    public static int sum(List<Integer> numbers, Conditional conditional) {
+        return numbers.stream()
+            .filter(conditional::test)
+            .reduce(0, Integer::sum);
+    }
 
     public static int sumAll(List<Integer> numbers) {
-        int total = 0;
-        for (int number : numbers) {
-            total += number;
-        }
-        return total;
+        // before        
+        // int total = 0;
+        // for (int number : numbers) {
+        //     total += number;
+        // }
+        // return total;
+
+        // after
+        return sum(numbers, new Conditional() {
+            @Override
+            public boolean test(Integer number) {
+                return true;
+            }
+        });
```

Comment:
익명 클래스보다 람다를 활용해 보시면 어떨까요?
```suggestion
        return sum(numbers, number -> true);
```